gramjs/tl/api.d.ts

```diff
@@ -5088,15 +5145,15 @@ export namespace Api {
     }
     export class ReceivedNotifyMessage extends VirtualClass<{
         id: int;
-        flags: int;
+        // flags: int;
```

Comment:
I think you're using an old version to generate these? 
This should not be commented and flags2 should be commented